### PR TITLE
Some change in login module

### PR DESCRIPTION
**Pull request by KingFarGrace**
+ Add these following dependencies to **pom.xml**:
  + MyBatis
  + Druid datasource
  + log4j
+ Add mapperScan annotation to **boost class**
+ Add some config for MyBatis and Druid to **application.yml**
+ Create database connection
+ Reorganize package **entity**
+ Add package **mapper** and a concreate mapper class/xml
+ Revise the logic of AccountServiceImpl and AccountController
+ Wrap UserLoginData instance into AccountResponse

### CODE REVIEW

File: back-end/managementplatform/src/main/java/team/sunrise/managementplatform/entity/UserLoginData.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+import java.util.Objects;
+
+public class UserLoginData {
+    private String uid;
+    private String username;
+    private String password;
+    private String telephone;
+    private String mail;
+    private boolean isNormal;
```
Comment (SilentWolf1997) [inline back-end/managementplatform/src/main/java/team/sunrise/managementplatform/entity/UserLoginData.java:11]: 其他权限都没有不就是普通用户了吗。。。这个会不会造成数据不一致

File: back-end/managementplatform/src/main/java/team/sunrise/managementplatform/service/AccountServiceImpl.java [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 @Service
 public class AccountServiceImpl implements AccountService {
+
+    @Autowired
+    UserLoginMapper userLoginMapper;
+
     @Override
-    public boolean login(LoginData data) {
+    public UserLoginData login(UserInputData data) {
         // TODO: implement login
```
Comment (SilentWolf1997) [inline back-end/managementplatform/src/main/java/team/sunrise/managementplatform/service/AccountServiceImpl.java:19]: 既然实现了就把todo也一起删了

File: back-end/managementplatform/src/main/java/team/sunrise/managementplatform/entity/UserLoginData.java (hunk 1 above)
Comment (KingFarGrace) [reply back-end/managementplatform/src/main/java/team/sunrise/managementplatform/entity/UserLoginData.java:11]: 删掉也行，就是如果做if else判断的话得多写点

File: back-end/managementplatform/src/main/java/team/sunrise/managementplatform/service/AccountServiceImpl.java (hunk 2 above)
Comment (KingFarGrace) [reply back-end/managementplatform/src/main/java/team/sunrise/managementplatform/service/AccountServiceImpl.java:19]: 1

**Review by SilentWolf1997 — APPROVED** [review]

**Pull request merged**